src/nested_pandas/nestedframe/utils.py

```diff
@@ -0,0 +1,34 @@
+def _ensure_spacing(expr) -> str:
+    """Ensure that an eval string has spacing"""
+    single_val_operators = ["+", "-", "*", "/", "%", ">", "<", "|", "&", "~", "="]  # omit "(" and ")"
+    check_for_doubles = ["=", "/", "*", ">", "<"]
+    double_val_operators = ["==", "//", "**", ">=", "<="]
+    expr_list = [*expr]
```

Comment:
I'm a bit confused by this line. Why do we do `[*expr]` instead of `list(expr)`? I believe we don't even need it to be a list: `expr_list[i]` is a single character string, `expr_list[i : i + 2]` is a slice over a string, which is also a string.

Reply:
I'm not sure why I opted to split the string out into a list, I was trying a few different solution methods and so perhaps this just stuck as I was working through it. You're correct in that we can just operate on this as a string, I've updated it to do so in the latest commit